client/signup/config/steps-pure.js

```diff
@@ -385,7 +385,38 @@ export function generateSteps( {
 
 		'site-information': {
 			stepName: 'site-information',
-			providesDependencies: [ 'siteTitle', 'address', 'email', 'phone' ],
+			providesDependencies: [ 'title', 'address', 'phone' ],
+			props: {
+				headerText: i18n.translate( 'Help customers find you' ),
+				informationFields: [ 'title', 'address', 'phone' ],
+			},
+		},
+
+		'site-information-title': {
+			stepName: 'site-information-title',
+			providesDependencies: [ 'title' ],
+			props: {
+				headerText: i18n.translate( "What's your business name?" ),
```

Comment:
Good point. I'll take a look.

Reply:
I think we could make this title generic, e.g., something like `"Tell us your site's name"` since if the site type is _"Business"_, the field's label will be _[What is the name of your business?](https://github.com/Automattic/wp-calypso/blob/master/client/lib/signup/site-type.js#L37)_ anyway.

Reply:
Making it generic sounds like a nice plan to me.